packages/kbn-optimizer/limits.yml

```diff
@@ -107,3 +107,4 @@ pageLoadAssetSize:
   osquery: 107090
   fileUpload: 25664
   banners: 17946
+  mapsEms: 26072
```

Comment:
Could `mapsLegacy` be reduced since this was broken out?

Running `node scripts/build_kibana_platform_plugins.js --update-limits` it looks like `legacyMaps` could be set to `87859` which is 5k above the current size.